modules/EnsEMBL/Web/Template/Legacy.pm

```diff
@@ -21,6 +21,7 @@ package EnsEMBL::Web::Template::Legacy;
 
 ### Legacy page template, used by standard HTML pages
 
+use EnsEMBL::Web::Utils::FileHandler qw(file_get_contents);
```

Comment:
Should we remove this import?

Reply:
I think so - the TmpMessage module should be reading the file, not the template.

Reply:
Removed the import